apps/native/app/src/screens/wallet-pass/wallet-pass.tsx

```diff
@@ -176,6 +183,11 @@ export const WalletPassScreen: NavigationFunctionComponent<{
   const theme = useTheme()
   const intl = useIntl()
   const [addingToWallet, setAddingToWallet] = useState(false)
+  const { walletPassDismissedInfoAlerts, setWalletPassInfoAlertDismissed } =
+    usePreferencesStore((state) => ({
+      walletPassDismissedInfoAlerts: state.walletPassDismissedInfoAlerts,
+      setWalletPassInfoAlertDismissed: state.setWalletPassInfoAlertDismissed,
+    }))
```

Comment:
Better to have this in separate lines, or use [useShallow](https://zustand.docs.pmnd.rs/hooks/use-shallow).